includes/class-blocks.php

```diff
@@ -13,6 +13,20 @@ public static function init() {
 		self::register_blocks();
 		\add_action( 'wp_enqueue_scripts', array( self::class, 'add_data' ) );
 		\add_action( 'enqueue_block_editor_assets', array( self::class, 'add_data' ) );
+		\add_action( 'load-post-new.php', array( self::class, 'handle_in_reply_to_get_param' ) );
+	}
+
+	/**
+	 * Enqueue the reply handle script if the inReplyTo GET param is set.
+	 */
+	public static function handle_in_reply_to_get_param() {
+		if ( ! isset( $_GET['inReplyTo'] ) ) {
```

Comment:
is it a good idea to have that in camel case?

Reply:
Happy to switch over, snake_case is more idiomatic in WP anyway :)